schema/shacl/SCHEMA_QUDT_NoOWL-v2.1.ttl

```diff
@@ -2302,16 +2304,10 @@ qudt:allowedUnitOfSystem
   a rdf:Property ;
   dcterms:description "This property relates a unit of measure with a unit system that does not define the unit, but allows its use within the system. An allowed unit must be convertible to some dimensionally eqiuvalent unit that is defined by the system."^^rdf:HTML ;
   dcterms:isReplacedBy qudt:applicableSystem ;
-  owl:deprecated true ;
   rdfs:isDefinedBy <http://qudt.org/2.1/schema/shacl/qudt> ;
   rdfs:label "allowed unit of system" ;
   rdfs:subPropertyOf qudt:isUnitOfSystem ;
-.
-qudt:applicableSystem
-  a rdf:Property ;
-  dcterms:description "This property relates a unit of measure with a unit system that may or may not define the unit, but within which the unit is compatible."^^rdf:HTML ;
-  rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;
-  rdfs:label "applicable system" ;
+  owl:deprecated true ;
```

Comment:
Use qudt:deprecated - this needs to be added as a property. Also will need to be on a property shape on qudt:Concept.

Reply:
OK, I think this is all fixed up and ready for review. There are quite a few changes (see the commits) but I have tested the whole set with my committee work data files that import QUDT and it behaves as expected:
![image](https://user-images.githubusercontent.com/1130189/230410836-ddcac467-bb43-4c10-b0c3-eb6a54b3b393.png)